rmq/rmq_integration_test.go

```diff
@@ -30,18 +30,19 @@ import (
 	"github.com/streadway/amqp"
 
 	"github.com/intelsdi-x/snap/control/plugin"
+	"github.com/intelsdi-x/snap/core"
 	"github.com/intelsdi-x/snap/core/ctypes"
 )
 
 // integration test
 func TestRmqIntegration(t *testing.T) {
-	mt := plugin.PluginMetricType{
-		Namespace_:          []string{"foo", "bar"},
+	mt := plugin.MetricType{
+		Namespace_:          core.NewNamespace([]string{"foo", "bar"}...),
```

Comment:
This should just be core.NewNamespace("foo","bar"). No need to make a slice and then split it up.